manual/parts/threads.tex

```diff
@@ -49,14 +49,16 @@ \subsection{Thread Creation}
 or by separately calling the \apifunc{seL4\_TCB\_Resume}{tcb_resume} method. Both of these methods
 place the thread in a runnable state.
 
-On the master kernel, this will result in the thread immediately being added to
+On the standard (non-MCS) kernel, this will result in the thread immediately being added to
```

Comment:
We should keep the terminology consistent. I checked -- the mentions in this paragraphs are the only ones that refer to "master" kernel, so if we get it correct here, we should be consistent. The origin of calling it "master kernel" is that MCS used to be a separate branch, which is no longer the case.

I'd propose `In non-MCS configurations of the kernel, this will in ..`

Reply:
I'll change this, if this is the official wording now

Reply:
Extended commit message to state that merge happened for version 11.0.0.